app/models/reject.rb

```diff
@@ -0,0 +1,5 @@
+class Reject < ActiveRecord::Base
+  belongs_to :user
+  has_many :users
+  validates_uniqueness_of :user_id, scope: :reject_id, message: "already rejected"
```

Comment:
Line is too long. [82/80]<br>Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.